change_analyzer/sequences_diff.py

```diff
@@ -185,13 +185,17 @@ def _get_attribute_value_based_on_node(page_root: ET.Element, node: str, attribu
 
     def _extract_diffs_as_dict(self, expected_xml: str, actual_xml: str) -> Dict:
         """Extract diffs as dictionary, using expected and actual approach"""
-        expected_root = ET.ElementTree(ET.fromstring(expected_xml)).getroot()
-        actual_root = ET.ElementTree(ET.fromstring(actual_xml)).getroot()
-
         diffs_as_dict = {
             'expected': {},
             'actual': {}
         }
+        # expected_xml and actual_xml can sometimes be nan, giving a TypeError
+        try:
+            expected_root = ET.ElementTree(ET.fromstring(expected_xml)).getroot()
+            actual_root = ET.ElementTree(ET.fromstring(actual_xml)).getroot()
+        except TypeError:
+            return diffs_as_dict
+

```

Comment:
Wouldn't it be better to instead of returning np.nan return empty "<?xml version="1.0" encoding="UTF-8"?>". That will make the code better in several places.